src/xb-silo-query.c

```
@@ -48,39 +48,41 @@ xb_silo_query_node_matches(XbSilo *self,
+						    &predicate_bindings,
+						    predicate_bindings_idx);
+						predicate_bindings_idx++;
+					}
+				}
+				if (!xb_machine_run_with_bindings(machine,
+								  opcodes,
+								  &predicate_bindings,
+								  result,
+								  query_data,
+								  error))
+					return FALSE;
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If G_DEFINE_AUTO_CLEANUP_CLEAR_FUNC is a macro then please configure it.

(unknownMacro)